src/m365/spo/commands/tenant/tenant-applicationcustomizer-set.ts

```diff
@@ -161,6 +170,13 @@ class SpoTenantApplicationCustomizerSetCommand extends SpoCommand {
       });
     }
 
+    if (newClientSideComponentId !== undefined) {
+      formValues.push({
+        FieldName: 'TenantWideExtensionComponentId',
```

Comment:
We should first check if the new component id actually is an application customizer that can be deployed tenant wide. See the checks in the add command for reference sake.